src/plugins/timeline/pluginSpec.js

```diff
@@ -228,4 +228,122 @@ describe('the plugin', function () {
         });
     });
 
+    describe("The timestrip composition policy", () => {
+        it("allows composition for plots", () => {
+            const parent = {
+                "composition": [],
+                "configuration": {},
+                "name": "Some timestrip",
+                "type": "time-strip",
+                "location": "mine",
+                "modified": 1631005183584,
+                "persisted": 1631005183502,
+                "identifier": {
+                    "namespace": "",
+                    "key": "b78e7e23-f2b8-4776-b1f0-3ff778f5c8a9"
+                }
+            };
+            const testTelemetryObject = {
```

Comment:
Since you use this object twice (minus the type), maybe move it up in scope, then, just add the type key in each test, I think that will make it more clear what's going on as well as decluttering.